v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-tcn/auto-ml-forecasting-demand-tcn.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "    training_pipelin_run = experiment.submit(training_pipeline)\n",
+    "    training_pipelin_run.wait_for_completion(show_output=False)\n",
+    "    IS_TCN_MODEL_TRAINED = True\n",
+    "else:\n",
+    "    from azureml.train.automl.run import AutoMLRun\n",
+    "    from azureml.pipeline.core.run import PipelineRun\n",
+    "\n",
+    "    PIPELINE_RUN_ID = (\n",
+    "        \"<Replace with the PipelineRunId used for training the best TCN model>\"\n",
+    "    )\n",
+    "    training_pipelin_run = PipelineRun(experiment=experiment, run_id=PIPELINE_RUN_ID)\n",
+    "    print(f\"Using previously trained model. Pipeline run ID: {PIPELINE_RUN_ID}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Train the baseline model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use Naive model as our baseline. To train it, we kick off another automl experiment with the following settings. Please note that we disabled DNN (`enable_dnn=False`), added the Naive model to the allowed models list, and set the number of iterations to 1 since we only interested in training one specific model. To reduce the training time, we set the number of cross validations to 2. Read the following [link](https://learn.microsoft.com/en-us/azure/machine-learning/v1/how-to-auto-train-forecast-v1#training-and-validation-data) for more information on this topic. Note that we did not use the cross validations settings for the TCN model training because such models typically require validation set that are substantially longer than a small multiple of a forecast horizon.\n",
+    "\n",
+    "The only `AutoMLConfig` settings you might consider changing are the `experiment_timeout_hours` and `allowed_models`. You might want to increase the experiment timeout if your data has lots of unique time series. The allowed model list can be modified to refect a different choice of the baseline model and can be selected from the supported [forecasting models](https://learn.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.forecasting) and [regression models](https://learn.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.regression)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_config = AutoMLConfig(\n",
+    "    task=\"forecasting\",\n",
+    "    debug_log=\"baseline.log\",\n",
+    "    primary_metric=\"normalized_root_mean_squared_error\",\n",
+    "    experiment_timeout_hours=1,\n",
+    "    iterations=1,\n",
+    "    training_data=train_dataset,\n",
+    "    label_column_name=target_column_name,\n",
+    "    compute_target=compute_target,\n",
+    "    enable_early_stopping=True,\n",
+    "    n_cross_validations=2,\n",
```

Comment:
Like commented in the previous PR, can we add some anotation that TCN doesn't actually use n_cross_validations? I know that the problem that TCN uses the parameter n_cross_validations will only be resolved completely after the refactoring of the code based, I'm a little worried about that without some explanation the parmaeter here would cause some confusion for users.